docs/Api/CatalogApi.md

```diff
@@ -48,7 +48,7 @@ try {
 
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
- **body** | [**\SquareConnect\Model\BatchDeleteCatalogObjectsRequest**](../Model/BatchDeleteCatalogObjectsRequest.md)| An object containing the fields to POST for the request.  See the corresponding object definition for field details. |
+ **body** | [**\SquareConnect\Model\BatchDeleteCatalogObjectsRequest**](../Model/\SquareConnect\Model\BatchDeleteCatalogObjectsRequest.md)| An object containing the fields to POST for the request.  See the corresponding object definition for field details. |
```

Comment:
This prefix on the link is wrong. The url does not exist. Check the files under: https://github.com/square/connect-php-sdk/tree/d7c597576d78d4236a5980ff374c14111ce1c2a5/docs/Model. 
I believe you don't need to add `\SquareConnect\Model\` to any link